libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/NotificationConverterCommon.java

```diff
@@ -278,15 +280,16 @@ public String getFullyQualifiedEmailAddress(String userName) {
         return userName.replaceAll(userDomainPrefixRegex, "") + AT + emailDomainTo;
     }
 
-    public String getWorkflowUrl() {
-        return workflowUrl == null ? "" : workflowUrl;
+    public String getAdminWorkflowUrl() {
+        return adminWorkflowUrl == null ? "" : adminWorkflowUrl;
     }
 
-    public String getWorkflowUrl(String domainName) {
-        if (workflowUrl == null) {
+
+    public String getDomainWorkflowUrl(String domainName) {
+        if (domainWorkflowUrl == null) {
```

Comment:
this is probably the wrong check since domainWorkflowUrl will never be null since we have:

domainWorkflowUrl = athenzUIUrl + "/workflow/domain";

so we should probably check on the athenUIUrl being null or not.